art-decisions/proposals/2022-10-26-standardize-on-poetry.md

```diff
@@ -0,0 +1,44 @@
+# Title
+
+All python projects in Arcalot should standardize on the use of
+[poetry](https://python-poetry.org/) for dependency management and packaging.
+This standard should be built into documentation and contribution guidelines,
+included in any templates, and should be checked automatically by CI.
+
+## Benefits
+
+From @mfleader:
+
+`requirements.txt` in no way defines how a python project can be built and
+published as a package. From [PyPA](https://www.pypa.io/en/latest/) on python
+project configuration, the `pyproject.toml` is equivalent to `requirements.txt
+&& setup.py`. If we're requiring that plugins to be publishable, and `setup.py`
```

Comment:
typo, change to: "If we're requiring plugins to be publishable"

Reply:
fixed